pkg/gui/controllers/helpers/commits_helper.go

```diff
@@ -220,9 +220,12 @@ func (self *CommitsHelper) addCoAuthor(suggestionFunc func(string) []*types.Sugg
 		Title:               self.c.Tr.AddCoAuthorPromptTitle,
 		FindSuggestionsFunc: suggestionFunc,
 		HandleConfirm: func(value string) error {
-			commitMessage := self.JoinCommitMessageAndDescription()
-			coAuthorString := commitMessage + "\nCo-authored-by: " + value
-			self.SetMessageAndDescriptionInView(coAuthorString)
+			commitDescription := self.getCommitDescription()
+			if len(commitDescription) != 0 {
+				commitDescription += "\n\n"
```

Comment:
Here we could use the same smartness as above, and check if the previous line is also a "Co-authored-by" line, and only add one line feed in that case.